src/autocomplete.ts

```diff
@@ -360,6 +361,15 @@ export default class Autocomplete {
         }
     }
 
+    /**
+     * remove all selected options
+     */
+    removeAllSelected() {
+        for (let i = (this.selected.length -1); i >=0; i -= 1) {
+            this.removeEntryFromSelected(this.selected[i]);
+        }
+    }

```

Comment:
Why go through the list backwards?

Reply:
Because on each `removeEntryFromSelected`, `this.selected` is updated (one entry is deleted) and so its length changes : if we start at 0, soon we have errors because the target items don't exist.